### PR TITLE
Fix softmax symbolic

### PR DESCRIPTION
**Pull request by bddppq**
https://github.com/pytorch/pytorch/commit/ba64724aeea8ad5d4b50cd1154fca5a011618333

@ezyang @soumith @jamesr66a

### CODE REVIEW

**Comment by bddppq** [comment]
segfaults stack trace can be found here:
https://ci.pytorch.org/jenkins/job/onnx-fb-universe-builds/job/py2-gcc5-ubuntu16.04/19374/console

**Comment by ezyang** [comment]
@jamesr66a I wonder why the segfault didn't happen when you ran the tests locally. Do you know?

**Comment by jamesr66a** [comment]
@ezyang I didn't realize you had merged the PR, I saw the CI failures and was going to fix it :p
